conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -17,8 +17,12 @@ dependencies:
 - imagecodecs>=2021.6.8
 - ipython
 - lazy-loader>=0.4
+<<<<<<< HEAD
 - libnvimgcodec-dev>=0.6.0,<0.7.0
 - libnvimgcodec>=0.6.0,<0.7.0
+=======
+- libnvimgcodec-dev=0.6.0
+>>>>>>> d3e322e (feat: Add comprehensive nvImageCodec installation verification script)
```

Comment:
Looks like some merge conflicts were committed. Fortunately these files are autogenerated

So to fix this, would just run [`rapids-dependency-file-generator`]( https://github.com/rapidsai/dependency-file-generator ) and commit the changes